test cases/common/98 subproject subdir/subprojects/sub_implicit/meson.build

```diff
@@ -10,4 +10,4 @@ sub_implicit_provide2_dep = dep
 # This one is not overridden but the wrap file tells the variable name to use.
 glib_dep = dep
 
-assert(get_option('opt') == 'overriden')
\ No newline at end of file
+assert(get_option('opt') == 'overridden')
```

Comment:
I try not to mess with missing newlines at eof even though I (and Git/GitHub) am not a fan

Reply:
Purge missing newlines with fire. :P